src/dev/MAX31855.cpp

```diff
@@ -1,27 +1,69 @@
 #include <EVT/io/SPI.hpp>
 #include <dev/MAX31855.hpp>
+#include <utility>
 
 using namespace EVT::core::IO;
 
 namespace TMU::DEV {
 
 MAX31855::MAX31855(SPI& spi, uint8_t device) : spi(spi), device(device) {}
 
-uint16_t MAX31855::readTemp() {
-    uint16_t temp;
+MAX31855::MaxStatus MAX31855::readTemp(uint16_t& temp) {
+    uint8_t lastDataBit = 0;
+    uint16_t returned_temp = 0;
```

Comment:
This name makes it a bit clearer that the data is temperature and more
```suggestion
    uint16_t returned_data = 0;
```